test/ngResource/resourceSpec.js

```diff
@@ -1443,6 +1632,7 @@ describe('cancelling requests', function() {
       });
 
       var creditCard = CreditCard.get();
+      $rootScope.$apply();
```

Comment:
Yes. I believe it's because we're putting a promise between the GET request and the invocation to $http, which has to resolve first. We can wrap creating the requestInterceptor promise into a conditional though, which would remove the need for this line.